pkg/kubectl/describe.go

```diff
@@ -545,7 +545,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 			}
 		}
 		describeVolumes(pod.Spec.Volumes, out, "")
-		fmt.Fprintf(out, "QoS Class:\t%s\n", qos.GetPodQOS(pod))
+		fmt.Fprintf(out, "QoS Tier:\t%s\n", qos.GetPodQOS(pod))
```

Comment:
Yeah, we should probably make the tests more tolerant, or ideally not test changing output (though that's hard).

In the past we've done things like https://github.com/jfrazelle/kubernetes/blob/6bdd53ca6114dd10af854d5a2c8c8a8108bf05fb/test/e2e/dns.go#L183 to branch logic based on the version under test. Something like that may be applicable here, though it's kinda gross, too.

cc @kubernetes/sig-testing